MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 include *.txt
 include *.md
-recursive-include cf_units *.py *.py *.pxd *.pyx
+recursive-include cf_units *.py *.py *.pxd *.pyx *.c
```

Comment:
@pelson Not done in this PR, but there is a duplicated `*.py`... we only need it once, right?